src/Robo/Commands/Artifact/AcHooksCommand.php

```diff
@@ -71,6 +71,30 @@ public function postCodeUpdate($site, $target_env, $source_branch, $deployed_tag
     }
   }
 
+  /**
+   * Execute sql-sanitize against a database hosted in AC Cloud.
+   *
+   * This is intended to be called from db-scrub.sh cloud hook.
+   *
+   * @param string $site
+   *   The site name. E.g., site1.
+   * @param string $target_env
+   *   The cloud env. E.g., dev
+   *
+   * @command artifact:ac-hooks:db-scrub
+   *
+   * @throws \Exception
+   */
+  public function dbScrub($site, $target_env) {
+    $this->taskDrush()
+      ->drush("sql-sanitize --sanitize-password=\"$(openssl rand -base64 32)\" --yes")
```

Comment:
Since we are in PHP, I'd prefer to generate the random password in PHP. See https://github.com/acquia/blt/blob/9.x/src/Robo/Commands/Setup/DrupalCommand.php#L31